minio/helpers.py

```diff
@@ -255,7 +255,7 @@ def md5sum_hash(data):
     if data is None:
         return None
 
-    hasher = hashlib.md5()
+    hasher = hashlib.new("md5", usedforsecurity=False)
```

Comment:
Please add a comment here @BOPOHA

Reply:
I hope this will be enough.

Reply:
Just point to the issue instead of descriptive comments. Something like 
```
# indicate md5 hashing algorithm is not used in a security context.
# Refer https://bugs.python.org/issue9216 for more information.
```